views/eApostilles/applicationType.ejs

```diff
@@ -0,0 +1,150 @@
+<% pageTitle="Choose a service - Get a document legalised – GOV.UK" %>
+
+    <div class="inner_header no-user-signed-in">
+        <%if(back_link){%>
+            <a href="<%= back_link%>" class="back-to-previous govuk-link">Back</a>
+        <%}%>
+        <%- partial ('../partials/inner-header.ejs')%>
+    </div>
+
+    <%if (error_report) { %>
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+            data-module="govuk-error-summary">
+            <h2 class="govuk-error-summary__title" id="error-summary-title">
+                There is a problem
+            </h2>
+            <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                    <li>
+                        <a href="#no-service-chosen-error">You must select a service type.</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    <%}%>
+
+    <h1 class="heading-large">Which service would you like?</h1>
```

Comment:
If you have the electronic flag enabled in the database, do you only get the option of e-App or standard journeys now? What about Premium or Drop-off?

Reply:
With the e-App flag only premium and not drop off, but the MVP will only go out to a handful of notaries before we make more changes and open it up to more notaries so the drop-off might be added later on.

Reply:
Speaking with James Richards today on our standup, they are expecting the drop off service to be there even for the beta so it looks like it needs to be enabled. Can you re-confirm this requirement?